eos/form-factors/analytic-b-to-pi_TEST.cc

```diff
@@ -248,22 +248,62 @@ class AnalyticFormFactorBToPiDKMMO2008Test :
                 TEST_CHECK_NEARLY_EQUAL( 0.2211, ff_no_rescale.FT_nlo_tw2( 10.0), 1. * eps);
 
                 // FT NLO, tw3
-                TEST_CHECK_NEARLY_EQUAL(-0.0833, ff_no_rescale.FT_nlo_tw3(-10.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL(-0.1570, ff_no_rescale.FT_nlo_tw3(- 1.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL(-0.1717, ff_no_rescale.FT_nlo_tw3(  0.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL(-0.1887, ff_no_rescale.FT_nlo_tw3(  1.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL(-0.2928, ff_no_rescale.FT_nlo_tw3(  5.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL(-0.6184, ff_no_rescale.FT_nlo_tw3( 10.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL(-0.0314, ff_no_rescale.FT_nlo_tw3(-10.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL(-0.0603, ff_no_rescale.FT_nlo_tw3(- 1.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL(-0.0665, ff_no_rescale.FT_nlo_tw3(  0.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL(-0.0740, ff_no_rescale.FT_nlo_tw3(  1.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL(-0.1220, ff_no_rescale.FT_nlo_tw3(  5.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL(-0.2879, ff_no_rescale.FT_nlo_tw3( 10.0), 1. * eps);
 
                 // fT form factor @ mu = 3.0
-                TEST_CHECK_NEARLY_EQUAL( 0.1685, ff_no_rescale.f_t(-10.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL( 0.2080, ff_no_rescale.f_t( -5.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL( 0.2515, ff_no_rescale.f_t( -1.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL( 0.2647, ff_no_rescale.f_t(  0.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL( 0.2790, ff_no_rescale.f_t(  1.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL( 0.3510, ff_no_rescale.f_t(  5.0), 1. * eps);
-                TEST_CHECK_NEARLY_EQUAL( 0.4904, ff_no_rescale.f_t( 10.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.1751, ff_no_rescale.f_t(-10.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.2171, ff_no_rescale.f_t( -5.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.2638, ff_no_rescale.f_t( -1.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.2781, ff_no_rescale.f_t(  0.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.2937, ff_no_rescale.f_t(  1.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.3728, ff_no_rescale.f_t(  5.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.5326, ff_no_rescale.f_t( 10.0), 1. * eps);
+            }
 
+            {
+                // Comparison with Blazenka's notebook underlysing the [DKKMO:2008A] results
```

Comment:
Since you opened a scope here, I guess `Parameters p = Parameters::Defaults();` is missing.

Reply:
Fixed. Moreover, the actuall form factor object itself was missing. Lazy conflict resolution on my part, I assume.